src/verifiers/documentStatus/tokenRegistryStatus/openAttestationEthereumTokenRegistryStatus.test.ts

```diff
@@ -110,32 +110,34 @@ describe("openAttestationEthereumTokenRegistryStatus", () => {
           network: "ropsten",
```

Comment:
there is no valid for v2 in this file

Reply:
L236?
https://github.com/Open-Attestation/oa-verify/pull/132/files#diff-7a41dabbcd29103a80790079392c006d1bcfd8222c200174ed29acd2ca8b4f63R236